exp/addons/internal/controllers/clusterresourceset_controller_test.go

```diff
@@ -426,6 +422,10 @@ metadata:
 			return false
 		}, timeout).Should(BeTrue())
 
+		// Must sleep here to make sure resource is created after the previous reconcile.
```

Comment:
TODO: check if this can be moved few lines above (essentially revert this diff)

Reply:
Yeah lets see if we leave the sleep on line 399 if the tests still work

Reply:
`make test` passed without this commit so dropped it